pkgs/test/default.nix

```diff
@@ -55,4 +55,6 @@ with pkgs;
   trivial-overriding = callPackage ../build-support/trivial-builders/test-overriding.nix {};
 
   writers = callPackage ../build-support/writers/test.nix {};
+
+  buildEnv = callPackage ../build-support/buildenv/test.nix {};
```

Comment:
This should probably be sorted alphabetically.

Reply:
Sorted it, while keeping cc related stuff at the top and keeping stuff that had no empty lines between them together.